hknweb/exams/models.py

```diff
@@ -1,30 +1,39 @@
 from django.db import models
 
 class Department(models.Model):
-    name = models.CharField(unique=True, max_length=4, null=True) #short form (e.g. 'EE')
-    subject = models.CharField(max_length=255) #long form (e.g. 'Electrical Engineering')
+    abbreviated_name = models.CharField(unique=True, max_length=4, null=True) #short form (e.g. 'EE')
+    long_name = models.CharField(max_length=255) #long form (e.g. 'Electrical Engineering')
+
+    def getAbbrName(self):
+        return self.abbreviated_name
```

Comment:
This isn't Java. We don't need a getter for a property access. If we actually need to rename the underlying variable underneath, we can do that with `@property` later.

If you haven't figured this out I'm quite jaded from Java's verbosity.